### DISCUSSION TITLE
x-transition is not working on 3.2.3

### DISCUSSION THREAD

**Discussion by khaiknievel** [alpinejs/alpine#1914] [7. Bugs (archived)]
Added package via `yarn add alpinejs`
Followed simple example at https://alpinejs.dev/directives/transition. 
Seems like the css property opacity and transform is not being applied. 

Works in 3.2.2 via `yarn add alpinejs@3.2.2`

**Comment by KevinBatdorf** [comment]
I can confirm this. Here's a demo set up:

https://codepen.io/KevinBatdorf/pen/BaRqbpa?editors=1010

**Reply by KevinBatdorf** [reply]
Sorry @SimoTod but tagging you because i know this stuff interests you and I'm not sure i have time this weekend to debug it. Also @calebporzio I should tag you as well I suppose ;-)

**Reply by SimoTod** [reply]
@KevinBatdorf I don't know why but apparently this PR https://github.com/alpinejs/alpine/pull/1854 broke transitions.

**Reply by SimoTod** [reply]
Fixed here https://github.com/alpinejs/alpine/pull/1920

**Comment by klaasgeldof** [comment]
Yep, reported this 4 days ago: #1905
